### ISSUE TITLE
A11y violation in Navbar

### ISSUE THREAD

**Issue by admmasters** [zopaUK/react-components#705]
https://zrc.netlify.app/#/Components/Organisms/Navbar

There is a warning in lighthouse related to this component when used in an application:

Lists do not contain only <li> elements and script supporting elements (<script> and <template>).
Screen readers have a specific way of announcing lists. Ensuring proper list structure aids screen reader output.

![image](https://user-images.githubusercontent.com/335607/123056419-ced2af80-d3fe-11eb-82e1-b4885a036237.png)

**Comment by OlenaKashuba** [comment]
![image](https://user-images.githubusercontent.com/27281884/123057168-8cf63900-d3ff-11eb-94e4-7ff0b5b90653.png)
